security/middleware.py

```diff
@@ -528,37 +537,40 @@ class XFrameOptionsMiddleware(BaseMiddleware):
         <http://tools.ietf.org/html/rfc7034>`_
     """
 
-    OPTIONAL_SETTINGS = ('X_FRAME_OPTIONS', 'X_FRAME_OPTIONS_EXCLUDE_URLS')
+    OPTIONAL_SETTINGS = ("X_FRAME_OPTIONS", "X_FRAME_OPTIONS_EXCLUDE_URLS")
 
-    DEFAULT = 'deny'
+    DEFAULT = "deny"
 
     def __init__(self, get_response=None):
         super().__init__(get_response)
-        warnings.warn((
-            'An official middleware "{name}" is supported by Django. '
-            'Refer to {url} to see if its approach fits the use case.'
-        ).format(
-            name="XFrameOptionsMiddleware",
-            url=DJANGO_CLICKJACKING_MIDDLEWARE_URL))
+        warnings.warn(
```

Comment:
Would be nice if we can drop this too but I think this will require additional changes for us since the django middleware doesn't allow exclusion by URLs. Maybe we can make a followup ticket to look into that so we have one less thing to maintain

Reply:
I vote to leave this as is for now and revisit later.

Reply:
We can track this in a ticket and resolve this comment with a ref